01.fizzbuzz/fizzbuzz.js

```diff
@@ -0,0 +1,11 @@
+for (let number = 0; number <= 20; number++) {
+    if (number % 15 === 0) {
+        console.log("FizzBuzz");
+    } else if (number % 3 === 0) {
+        console.log("Fizz");
+    } else if (number % 5 === 0) {
+        console.log("Buzz");
+    } else
```

Comment:
else節だけ`{}`が付かないのはちょっと不自然なので、ここも`{}`で囲みましょう。

```suggestion
    } else {
```

Reply:
お恥ずかしい、ありがとうございます🙏